idunn/blocks/environment.py

```diff
@@ -36,26 +36,28 @@ class AirQuality(BaseBlock):
     measurements_unit: Optional[str] = None
 
     @classmethod
-    def from_es(cls, place, lang):
-        if not settings["BLOCK_AIR_QUALITY_ENABLED"]:
-            return None
-        if place.PLACE_TYPE != "admin":
-            return None
-        if place.get("zone_type") not in ("city", "city_district", "suburb"):
+    def from_es(cls, es_poi, lang):
+        if (
+            not settings["BLOCK_AIR_QUALITY_ENABLED"]
+            or es_poi.PLACE_TYPE != "admin"
+            or es_poi.get("zone_type") not in ("city", "city_district", "suburb")
+        ):
```

Comment:
Yeah it came from pylint which complained that parameters weren't consistent with methods we are overriding, I would agree that this can be confusing. I'll rename everything to place as you suggested :)